packages/react-components/react-tags/package.json

```diff
@@ -34,6 +34,7 @@
   "dependencies": {
     "@fluentui/react-theme": "^9.1.7",
     "@fluentui/react-utilities": "^9.8.0",
+    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
```

Comment:
_nit:_ Can you move this above `@fluentui/react-theme` in line `35` to keep alphabetical order?